### DISCUSSION TITLE
Is tax rate integer only?

### DISCUSSION THREAD

**Discussion by sdussaut** [duncanmcclean/simple-commerce#1108] [Help]
Hi Duncan!

I was looking to setup tax for my project with Simple Commerce and it seems Tax only accepts integer values for rates?

Am I missing something here? In France, we do have floating tax rates (namely 5.5% for a lot of products) :-)

Thanks for you help!

**Comment by duncanmcclean** [comment]
In theory, floating tax rates should be supported. If you're having issues with them, then it's a bug.

**Reply by sdussaut** [reply]
I was only a matter of a missing HTML attribute which blocked the form submission all together. I added it here https://github.com/duncanmcclean/simple-commerce/pull/1109
